### DISCUSSION TITLE
Does the knockd service stay on when you're playing?

### DISCUSSION THREAD

**Discussion by josejsarmento** [itzg/docker-minecraft-server#1631] [Q&A]
This question is related to the auto-pause function. When you're playing, you are constantly knocking on the server port, thus triggering the knockd service every time. Is this always happening? Does this not overwhelm the CPU?

**Comment by itzg** [comment]
Good question. It is always running; however, since Java edition communicates over TCP and the connections are long lived there should be minimal involvement from knockd. 

That's only me speculating though...so I'm curious to monitor it and/or let me know if you notice performance impact from it.
